demo/client/src/3d-demo.ts

```diff
@@ -124,6 +124,15 @@ interface MeshDescriptor {
     positions: glmatrix.vec2[];
 }
 
+function F32ArrayToMat4(array: Float32Array): mat4 {
+    const mat = glmatrix.mat4.create();
```

Comment:
I believe you can just remove this function and use `glmatrix.mat4.clone(array)` instead: http://glmatrix.net/docs/mat4.js.html#line60